linode/manager
# DBAAS1-79: Database Create Page

## Description
Adds the ability to create a Database Cluster in `databases/create`

## To-do
- [x] Final copy and contextual help

## How to test
- Test frontend validation and ensure errors surface
- Nodes should be auto-selected based on selected plan type
    - 1 GB plans: auto-select 1 Node and disable 3 Nodes option
    - Other plans: auto-select 3 Nodes
- Check the POST data and ensure it matches API spec

**How do I run relevant unit tests?**
`yarn test DatabaseCreate`
